contrib/babelfishpg_tsql/sql/sys_function_helpers.sql

```diff
@@ -10310,3 +10310,7 @@ AS 'babelfishpg_tsql', 'get_server_collation_oid';
 CREATE OR REPLACE FUNCTION sys.babelfish_get_pltsql_function_signature(IN funcoid OID)
 RETURNS text
 AS 'babelfishpg_tsql', 'get_pltsql_function_signature' LANGUAGE C;
+
+CREATE OR REPLACE FUNCTION sys.babelfish_datatype_hash_tables_initialised()
```

Comment:
creating a SQL function with upgrade implications just for testing is not good, we should try to look for other alternatives

Reply:
I think we already have a couple of functions for internal debugging/testing purpose, [ref](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_2_X_DEV/contrib/babelfishpg_tsql/sql/sys_function_helpers.sql#L10293), moreover why do we think there are upgrade implications of this?

Reply:
I agree with Surendra. Once we expose SQL and C functions, it is almost impossible to remove them. One suggestion would be the following:

First of all, we can remove the SQL function in the installation script and create this on-demand in the test file and remove it.

For the C function, I'd suggest to expose a more generic function, e.g., babelfish_integrity_checker, and then inside that we can add some specific functions freely.

Reply:
Yeah it also makes sense to create the function on demand, addressed it in new commit!